tests/extractors/api/tests/test_datasources.py

```diff
@@ -2,10 +2,10 @@
 
 import mwapi
 
-from ....datasources import revision_oriented as ro
-from ..datasources import (LastUserRevDoc, PageCreationRevDoc,
-                           PropertySuggestionDoc, RevDocById, UserInfoDoc)
-from ..extractor import Extractor
+from revscoring.datasources import revision_oriented as ro
+from revscoring.extractors.api.datasources import (LastUserRevDoc, PageCreationRevDoc,
+                                                   PropertySuggestionDoc, RevDocById, UserInfoDoc)
```

Comment:
Oof.  This is an unforeseen drawback to absolute imports.  Tabbing is gonna get crazy.